css/danti.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="danti.css">
+</head>
+<body>
+
+    <div class="contener">
+        <div class="nav_all">
+        <img class="icon_nav_p2" src="../img/karma-logo.svg"/>
```

Comment:
Remember every image need an `alt` attribute which should attempt to provide equivalent meaning for users of screen-readers and other assistive technologies. In this case, it's a logo, so useful alt text would be the name of the company, `Karma`.